src/scripts/main.js

```diff
@@ -1,3 +1,34 @@
 'use strict';
 
-// write code here
+const thead = document.querySelector('thead');
+const tbody = document.querySelector('tbody');
+
+function convertToNumber(currency) {
+  return Number(currency.split(',').join('.').slice(1));
+}
+
+thead.addEventListener('click', (e) => {
+  const title = e.target.textContent;
```

Comment:
It's a good practice to check if the clicked element is a column header. Otherwise, if a user clicks on a space between headers, an error will occur because the `textContent` and `cellIndex` properties are undefined for such elements. You can use the `nodeName` property to check if the clicked element is a `TH` element.